### PR TITLE
Test env

### PR DESCRIPTION
**Pull request by dandds**
This is the relevant code for the new test environment (https://www.pivotaltracker.com/story/show/162132078). 

I'll leave it a WIP until we decide how frequently the new site should be updated, since that might require more config.

### CODE REVIEW

**Comment by dandds** [comment]
@patricksmithdds I added a quick outline of what we did to add the new site but I'm sure I'm missing some things. Could you review it when you get a chance?

File: deploy/kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Adding a New Environment
+
+## New Config
+
+Add a new subfolder to this directory. You can copy `uat` or `tests`. You'll need to change any references to the previous environment name (i.e., `uat`) to your new environment name. This includes things like the k8s namespace and the subdomain for the new site.
+
+## New Load Balancers
+
+You need two new load balancers. Currently, these are managed in Rackspace. You will need one for the regular site and one for the auth domain. They should have all of the Kubernetes worker nodes attached. When attached to the LB for the main site domain, the nodes should point to port 32761, which is the port where all our NGINX ingress is managed. The auth LB nodes should point to a new port in the 32700 - 32799 range.
```
Comment (patricksmithdds) [inline deploy/kubernetes/README.md]: I think it's also important to note the new port chosen must match the value specified for the `NodePort` in the new yaml file.

**Comment by dandds** [comment]
@kristiandds or @ddsdevon I'm getting [auth failures every time](https://circleci.com/gh/dod-ccpo/atst/2108) for my service account for the new environment. Could you glance at this if you have time and see if you can tell what I'm missing? Specifically:

```
Error from server (Forbidden): deployments.apps "atst" is forbidden: User "system:serviceaccount:atat:atat-deployer" cannot get deployments.apps in the namespace "atat-test"
```

Does the atat-deployer service account associated with my new k8s namespace (atat-test) need its own bearer token? 

Relatedly, do you know where the creds are for things like `K8S_USER_TOKEN` in the `deploy/kubernetes/atst-update-deploy.sh` script? I can see they're in CircleCI but I can't get the values there. I'd like to try running the update locally to see if I hit the same auth errors.

**Comment by dandds** [comment]
Okay, this should be working. At midnight every day, it should do the following for our test.* environment:

- deploy current master
- destroy any existing sample data and reseed the database with our basic sample data

This is up-to-date with the config that's been applied to our k8s cluster.

It is admittedly hard to test this. I've watched it work a couple times now. If you want, you can push a commit to change the cron job in the CircleCI config to run soon after your commit and add this branch (`test-env`) to all the lists of filtered branches in the `nightly` jobs. Just make sure you delete the commit after.

**Pull request merged**